### ISSUE TITLE
Disable/Configure Exponential Backoff

### ISSUE THREAD

**Issue by abhijeet-branch** [veeqo/advanced-sneakers-activejob#33]
Currently we are trying to migrate to `advanced-sneakers-activejob` from resque. A lot of our jobs are already written keeping resque in mind. So we don't want to retry a lot of our jobs, atleast not that many times as per exponential backoff configuration.

Is there a way to disable the exponential backoff globally or limit through the number of retries? This is something which is becoming a blocker in our migration.

**Comment by sharshenov** [comment]
Hello,

There are a few options

1. Ignore errors on ActiveJob level (`discard_on StandardError` in `ApplicationJob`)
2. Use your own failure handler
   ```ruby
    AdvancedSneakersActiveJob.configure do |config|
      config.sneakers = {
        handler: MyHandler
      }
    end
    ```

**Comment by abhijeet-branch** [comment]
Thanks for the solution @sharshenov . I was able to implement second option. I implemented my Handler to push a message to another failure queue, I created a dummy job for the same.

So for the queue used for this failure jobs I want this queue to have a ttl for it's messages and I want it to be lazy-queue. Is there a possibility that I can define it to be on a Job level that queue should be lazy and should have a ttl associated to it.

```ruby
 AdvancedSneakersActiveJob.configure do |config|
      config.sneakers = {
        queue_options: {
          arguments: { 'x-queue-mode' => 'lazy', 'x-message-ttl' => 180 },
         }
       }
end
``` 

By doing this It makes it default configuration for all queues created. Is it possible to configure this on Job Level and pass arguments somehow during job declaration.
